bmi_index.js

```diff
@@ -0,0 +1,28 @@
+const formElement = document.querySelector(".bmi-form");
+
+const heightInput = document.querySelector("#height");
+const weightInput = document.querySelector("#weight");
+
+const bmiMessage = document.querySelector(".result");
+
+// const button = document.querySelector("button");
+
+// function calculate(heightInput, weightInput) {
+//   let result = heightInput.value / weightInput.value ** 2;
+//   console.log(result);
+// }
+
+formElement.addEventListener("submit", (event) => {
+  const bmi = (weightInput.value / heightInput.value ** 2) * 10000;
+  event.preventDefault();
+  bmiMessage.textContent = "Your BMI is:" + bmi;
+  if (+bmi < 18.5 || bmi > 30) {
```

Comment:
Why is there a `+` before the bmi?
But really clever to check here for  both conditions. I like it very much.

Reply:
wanted to be sure that bmi would end up a number. Removed it 👍